purchase.adoc

```diff
@@ -1,174 +1,300 @@
-## Purchase API
+= Purchase API
 
-### Service Description
-The purchase API provides information about purchases made through iZettle.
+== Description
+
+====
+Provides information about purchases made through iZettle.
+====
+
+=== Base URL
 
-### URL
 https://purchase.izettle.com
 
-### Scopes
-The Purchase API implements the following scopes:
+=== OAuth scopes
 
 - `READ:PURCHASE`
 
-=== Get purchase history
-Will include cash register information if applicable. Purchases up to three years old are available to fetch.
+'''
+
+==== Fetch purchase history
+
+Returns a list of purchases for organization.
```

Comment:
...for an organisation.
because iZettle still uses UK En as its corporate language.

Reply:
Will write merchant instead

Reply:
Since we use US in the code base

Reply:
> Will write merchant instead

It does sound better to use merchant. Now that I think of it, merchants are the users of the integrated services. Thanks for the heads-up!

Reply:
> Since we use US in the code base

Yeah, eventually the corporate language will change to the US en. Then let's keep it for now. We can ask Sarah Long later on the corporate language change.